pupil_src/shared_modules/calibration_routines/finish_calibration.py

```diff
@@ -12,23 +12,30 @@
 import os
 import numpy as np
 
-from . import calibrate
-from math_helper import *
-from file_methods import load_object, save_object
+import math_helper
+from file_methods import save_object
 
 from .optimization_calibration import bundle_adjust_calibration
-from .calibrate import find_rigid_transform
+from math_helper import find_rigid_transform
+from .data_processing import (
```

Comment:
Consider only using `import .data_processing` and later user `data_processing.match_data`. This makes it explicit that these functions are defined somewhere else and the prepended namespace also gives an additional hint at what the function is doing.

Alongside this change I also wonder if `data_processing` is the most descriptive name.